src/clients/facebook/FacebookAnalyticsClient.js

```diff
@@ -4,10 +4,10 @@ const Promise = require('promise');
 const request = require('request');
 
 const accessToken = process.env.FACEBOOK_AUTH_TOKEN;
-const apiHost = process.env.FACEBOOK_API_HOST || 'graph.facebook.com';
+const apiHost = process.env.FACEBOOK_API_HOST || 'https://graph.facebook.com';
```

Comment:
The `host` part in the variable name seems misleading to me; may I suggest 'const baseUrl' maybe as a followup PR at some point?